Makefile

```diff
@@ -4,4 +4,13 @@ proto:
 	buf generate
 	buf format -w
 	
+	# inject classification tags (see: https://github.com/hashicorp/go-eventlogger/tree/main/filters/encrypt)
+	@protoc-go-inject-tag -input=./github.com.hashicorp.go.kms.wrapping.v2.types.pb.go
+	
 .PHONY: proto
+
+
+.PHONY: tools
+tools: 
+	go install github.com/favadi/protoc-go-inject-tag
```

Comment:
Does this work? I think we should explicitly version it still.

Reply:
yes, it works.  happy to pin it.